src/graphql/data/nft/Collection.ts

```diff
@@ -0,0 +1,132 @@
+import graphql from 'babel-plugin-relay/macro'
+import { Trait } from 'nft/hooks/useCollectionFilters'
+import { GenieCollection } from 'nft/types'
+import { loadQuery, usePreloadedQuery } from 'react-relay'
+
+import RelayEnvironment from '../RelayEnvironment'
+import { CollectionQuery } from './__generated__/CollectionQuery.graphql'
+
+const collectionQuery = graphql`
+  query CollectionQuery($address: String!) {
+    nftCollections(filter: { addresses: [$address] }) {
+      edges {
+        cursor
+        node {
+          bannerImage {
+            url
+          }
+          collectionId
+          description
+          discordUrl
+          homepageUrl
+          image {
+            url
+          }
+          instagramName
+          isVerified
+          name
+          numAssets
+          twitterName
+          nftContracts {
+            address
+            chain
+            name
+            standard
+            symbol
+            totalSupply
+          }
+          traits {
+            name
+            values
+            stats {
+              name
+              value
+              assets
+              listings
+            }
+          }
+          markets(currencies: ETH) {
+            floorPrice {
+              currency
+              value
+            }
+            owners
+            totalVolume {
+              value
+              currency
+            }
+            listings
+            volume(duration: DAY) {
+              value
+              currency
+            }
+            volumePercentChange(duration: DAY) {
+              value
+              currency
+            }
+            floorPricePercentChange(duration: DAY) {
+              value
+              currency
+            }
+          }
+        }
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+        hasPreviousPage
+        startCursor
+      }
+    }
+  }
+`
+export function useCollectionQuery(address: string): GenieCollection | undefined {
+  const collectionQueryReference = loadQuery<CollectionQuery>(RelayEnvironment, collectionQuery, {
+    address,
+  })
+  const queryData = usePreloadedQuery<CollectionQuery>(collectionQuery, collectionQueryReference)
```

Comment:
I don't think `load` and `usepreload` should be used like this. iirc `loadQuery` should be happening higher up the component tree once we know the query might need to run and `usePreloadedQuery` should be happening here on its own at the last possible time.

Reply:
I was following the standard set by the token queries but instead replaced `useLazyLoadedQuery` with `usePreloadedQuery` as recommended by: https://relay.dev/docs/v12.0.0/api-reference/use-lazy-load-query/

Reply:
Yeah it's def better to use `usePreloadedQuery` and I'd love for you to implement that pattern in this PR, but the way you've implemented it here is (I think!) a more verbose way to accomplish `useLazyLoadedQuery`. If you look at the query preloading docs: https://relay.dev/docs/v12.0.0/api-reference/use-preloaded-query/

the example is trying to get users to do `useQueryLoader` with a query (in this case `CollectionQuery`) to get an instance of `loadQuery<CollectionQuery>` that can be called by user action ahead of the render loop that actually needs that data. The code that does that in the linked example is: 
```
<Button
      onClick={() => loadQuery({id: '4'})}
      ...
```

So I think a better way to accomplish this would be to use `useQueryLoader` directly in the component containing a link/button/card whatever that is going to route a user to the page that needs data from `CollectionQuery`. (I don't know of our page design **actually has** such an opportunity to intelligently pre-load a query, but that's the intended usage.) If there's no such opportunity to pre-load the query ahead of its usage (instead of alongside it) then the correct implementation would (I think) be `useLazyLoadedQuery`.

Reply:
I don't think it really makes sense to query for all potential collection queries a user can go to. I will switch usage to useLazyLoaded.

Reply:
I thiiink the right solution would be to load it on click of the collection a user is trying to navigate to